### PR TITLE
[enterprise-4.4] Add known issues about intermittent connectivity

### PR DESCRIPTION
**Pull request by jboxman**
- https://bugzilla.redhat.com/show_bug.cgi?id=1791162
- https://bugzilla.redhat.com/show_bug.cgi?id=1809665

### CODE REVIEW

**Comment by jboxman** [comment]
@codyhoag, what's an appropriate placement for this in the list? Thanks.

File: release_notes/ocp-4-4-release-notes.adoc [hunk 1]
```diff
@@ -2030,6 +2030,14 @@ In the table below, features are marked with the following statuses:
 [id="ocp-4-4-known-issues"]
 == Known issues
 
+When upgrading to a new {product-title} z-stream release, connectivity to the API server might be interrupted as nodes are upgraded, causing API requests to fail.
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1791162[*BZ#1791162*])
```
Comment (codyhoag) [inline release_notes/ocp-4-4-release-notes.adoc:2035]: We only add the BZ link using a `+` for entries that have steps/bullets. This (and the one after) can be added to the end of the sentence.

File: release_notes/ocp-4-4-release-notes.adoc [hunk 2]
```diff
@@ -2030,6 +2030,14 @@ In the table below, features are marked with the following statuses:
 [id="ocp-4-4-known-issues"]
 == Known issues
 
+When upgrading to a new {product-title} z-stream release, connectivity to the API server might be interrupted as nodes are upgraded, causing API requests to fail.
```
Comment (codyhoag) [inline release_notes/ocp-4-4-release-notes.adoc:2033]: Add both known issues as bullet points for consistency.

File: release_notes/ocp-4-4-release-notes.adoc [hunk 3]
```diff
@@ -2030,6 +2030,14 @@ In the table below, features are marked with the following statuses:
...
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1791162[*BZ#1791162*])
+
+When upgrading to a new {product-title} z-stream release, connectivity to routers might be interrupted as router pods are updated. For the duration of the upgrade, some applications might not be consistently reachable.
```
Comment (codyhoag) [inline release_notes/ocp-4-4-release-notes.adoc:2037]: s/pods/Pods

**Review by codyhoag — COMMENTED** [review]
I added a couple minor comments. We don't have any concrete rules around ordering of the known issues. I typically add new entries to the end, so anybody closely monitoring the list can easily find new known issues logged in the release notes. Everything else LGTM! Thanks!

**Comment by jboxman** [comment]
@codyhoag, thanks!

**Pull request merged**